controllers/config/manager/manager.yaml

```diff
@@ -23,6 +23,9 @@ spec:
     metadata:
       annotations:
         kubectl.kubernetes.io/default-container: manager
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /metrics
+        prometheus.io/port: "8080"
```

Comment:
Do we need the `prometheus.io/scheme: http` annotation, or is that the default?

Reply:
I believe it is the default. But then again, the path is also the default, so having one and not the other is a little inconsistent.